ush/machine/cheyenne.yaml

```diff
@@ -29,4 +31,4 @@ platform:
   FIXshp: /glade/p/ral/jntp/UFS_SRW_App/develop/NaturalEarth
 data:
   ics_lbcs:
-    FV3GFS: /glade/p/ral/jntp/UFS_CAM/COMGFS/gfs.${yyyymmdd}/${hh}
+    FV3GFS: /glade/collections/rda/data/ds084.1/${yyyy}/${yyyymmdd}
```

Comment:
@padhrigmccarthy, is this path for all users or your team? I wonder if this new path  works for the WE2E tests. Since I don't have access to Cheyenne, I'd like to hear from other code managers.

Reply:
While the WE2E tests for both Cheyenne Intel and Cheyenne GNU run without issue with this change, I think that we should point to the directories maintained by `ufssrw`, until such time that they are transitioned to EPIC.  Further input would be appreciated.

Reply:
Happy to use the default location.